tests/yam/agama/agama_headless.pm

```diff
@@ -0,0 +1,56 @@
+## Copyright 2025 SUSE LLC
```

Comment:
Avoid always to create Yet Another test module for the same thing and explore possibilities using the existing modules. We probably need a new module for new things, which are those checks for knowing that there is not desktop running or browser running.
We could add the install command to the import module (I think we will refactor this later so it could be a possibility) the other possibility is just to create a simple test module with that command.
Then you just need to monitoring the installation with normal agama module test module and the puppteer test that just check a running installation, that should work.

Reply:
For the current puppeteer test check the congratulation page for installation finished, should change the puppeteer test to check log in console for headless test.

Reply:
should behave like a remote installation even in qemu: https://openqa.suse.de/tests/17430474#step/agama/30, using test_running_installation

Reply:
OK, I'd move the few check for multi-user, desktop,browser to a new module `agama_headless_check` before import_json_file, and issue 'agama install` in import_json_file and use 'test_running_installation' in agama to do it.  Thanks

Reply:
For naming you can follow same pattern, under yam/ folder there are multiple validation, this one could be validate_headless.pm